src/Tests/Microsoft.Diagnostics.Monitoring.ExecuteActionApp/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Threading;
+
+namespace Microsoft.Diagnostics.Monitoring.ExecuteActionApp
+{
+    internal class Program
+    {
+        public static int Main(string[] args)
+        {
+            string testType = args[0];
```

Comment:
Consider some argument validation to make it easier to figure out failures

Reply:
What did you have in mind? In theory, since for each test we're manually setting the args, shouldn't we know that they'll all be valid?

Reply:
For example:
- Maybe the default case of the switch should throw an exception stating that the test type is not one of the known ones.
- Maybe extract the remaining args into a new array that the test types can look instead of looking at the original `args` parameter. This allows us to change the ordering of the arguments passed on the command line without having to rewrite all of the test cases.
- Each test type should valid that its args length is exactly what it should expect and throw an exception if it is different.